app/src/main/java/at/roteskreuz/stopcorona/model/entities/configuration/ApiConfiguration.kt

```diff
@@ -46,6 +49,25 @@ data class ApiConfiguration(
     }
 }
 
+@JsonClass(generateAdapter = true)
+data class ApiExposureConfiguration (
+    @field:Json(name = "minimum_risk_score")
+    val minimumRiskScore: Int,
+    @field:Json(name = "daily_risk_threshold")
+    val dailyRiskThreshold: Int,
+    @field:Json(name = "attenuation_duration_thresholds")
+    val attenuationDurationThresholds: List<Int>,
+    @field:Json(name = "attenuation_level_values")
+    val attenuationLevelValues: List<Int>,
+    @field:Json(name = "days_since_last_exposure_level_values")
+    val daysSinceLastExposureLevelValues: List<Int>,
+    @field:Json(name = "duration_level_values")
```

Comment:
It would be nice if you have some more info about the content to write it there as a KDoc to each field.

Reply:
@falkorichter